src/fvOptions/crossFlowTurbineALSource/crossFlowTurbineALSource.C

```diff
@@ -210,7 +210,7 @@ void Foam::fv::crossFlowTurbineALSource::createBlades()
         bladeSubDict.add
         (
             "nVelocitySamples",
-            coeffs_.lookupOrDefault("nVelocitySamples", 8)
+            coeffs_.lookupOrDefault("nVelocitySamples", 20)
```

Comment:
Actually, I need to double check, but these values should come straight from the blade subdict automatically, which would allow their use with struts as well.

Reply:
At the moment, the default value is set on 3 different places, either in actuatorLineSource.C, in crossFlowTurbineALSource.C or in axialFlowTurbineALSource.C. Maybe one default position should be sufficient. I had to add them in crossFlowTurbineALSource.C to allow the parameters to be modified from the input files. As this connection isn't added for the struts yet, only the default values from actuatorLineSource.C should apply for the struts, which is a radius of 0, which turns off this modification.

Reply:
I am going to submit a PR right now where you would put the values in the blade subdictionary rather than the top level turbine dictionary, so you can give that a test with your cases.